theme/scripts/patronSlider.ts

```diff
@@ -0,0 +1,161 @@
+import * as rangeSlider from "nouislider";
+import getNextSibling from "./utils/getNextSibling";
+
+export default (function () {
+  const PatronSlider = {
+    SELECTORS: {
+      slider: "[data-patron-slider]",
+      handle: ".noUi-handle",
+      pips: ".noUi-marker",
+      pipsValue: ".noUi-value",
+      patronTierLink: "[data-patron-tier-link]",
+      patronTierDescription: "[data-patron-tier-description]",
+      patronTierName: "[data-patron-tier-name]",
+    },
+    START_VALUE: 15,
+    MONTHLY: "Monthly",
+    PATRON_TIERS: {
+      "5": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7795091",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, member-only events, and our archived talks earlier than the public.",
+        name: "Community Member",
+      },
+      "10": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7795092",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, member-only events, and our archived talks earlier than the public. You'll also support our ongoing programming at Index.",
+        name: "Supporter Member",
+      },
+      "15": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7849151",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, member-only events, and our archived talks earlier than the public. You'll also support our ongoing programming at Index.",
+        name: "Supporter Member+",
+      },
+      "20": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7836923",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, member-only events, and our archived talks earlier than the public. You'll also help sustain our programming at Index.",
+        name: "Sustaining Member",
+      },
+      "25": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7849152",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, member-only events, and our archived talks earlier than the public. You'll also help sustain our programming at Index.",
+        name: "Sustaining Member+",
```

Comment:
Sustaining Patron+
You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also help sustain our programming at Index.